projects/lookup/src/modules/lookup/lookup-show-more-async-modal.component.html

```diff
@@ -0,0 +1,164 @@
+<sky-modal>
+  <sky-modal-header>
+    <ng-container *ngIf="context.userConfig?.title; else defaultTitle">
+      {{ context.userConfig.title }}
+    </ng-container>
+  </sky-modal-header>
+  <sky-modal-content>
+    <sky-wait [isWaiting]="isSearching"></sky-wait>
```

Comment:
Should the wait indicator be covering up the search bar as well? Is there any reason to prevent a user from typing in a search while data is being loaded?

Reply:
Leaving the search bar enabled while results are being loaded could result in multiple simultaneous queries being executed on the consumer's web service/database, which could lead to performance issues. Having said that, the same issue exists in an autocomplete "search-as-you-type" scenario. Let's leave it for now and bring it up with design at some point to see if we need to change the behavior.